### PR TITLE
Create Spanish verification email

### PR DESCRIPTION
**Pull request by rickreyhsig**
https://www.pivotaltracker.com/story/show/187048350

Screen Shot 2024-02-15 at 3 59 45 PM

### CODE REVIEW

File: app/mailers/verification_code_mailer.rb [hunk 1]
```diff
@@ -6,11 +6,11 @@ def with_code
     @locale = params[:locale]
     @verification_code = params[:verification_code]
     attachments.inline['logo.png'] = service.email_logo
-    if @service_type == :statefile
-      @subject = I18n.t("messages.verification_code_subject_with_service_name", service_name: @service_name, locale: @locale)
-    else
-      @subject = I18n.t("messages.default_subject_with_service_name", service_name: @service_name, locale: @locale)
-    end
+    @subject = if @service_type == :statefile
```
Comment (rickreyhsig) [inline app/mailers/verification_code_mailer.rb:9]: Rubocop suggested this styling

**Review by mpidcock — APPROVED** [review]

File: app/mailers/verification_code_mailer.rb [hunk 2]
```diff
@@ -6,11 +6,11 @@ def with_code
...
+                 I18n.t('messages.verification_code_subject_with_service_name', service_name: @service_name, locale: @locale)
```
Comment (mpidcock) [inline app/mailers/verification_code_mailer.rb:10]: Where is `@locale` coming from? I think part of this story is to collect the locale from the language setting when the user is on on questions/contact-preference and keep that as the stored locale/language.

Comment (rickreyhsig) [reply app/mailers/verification_code_mailer.rb:10]: Rails parses the URL and adds it to the params hash in `params[:locale]` line 6 of this file.

Comment (rickreyhsig) [reply app/mailers/verification_code_mailer.rb:10]: @mpidcock 👆🏼

Comment (mpidcock) [reply app/mailers/verification_code_mailer.rb:10]: oh nice, it's already there 👍

**Pull request merged**